components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1026,6 +1031,14 @@ boolean isImplicitGrantEnabled() {
         return supportedTokenTypes;
     }
 
+    /**
+     * Return the default token type.
+     */
+    public String getDefaultTokenType() {
+
+        return "Default";
```

Comment:
Shall we have a constant for this?

Reply:
How about introducing this getDefaultTokenType() the OAuthServerConfiguration class and using here?

Reply:
Reused the already defined constant in OAuthServerConfiguration. WDYT?